### ISSUE TITLE
isSelected within option causes Meteor error

### ISSUE THREAD

**Issue by vaughnkoch** [raix/Meteor-handlebar-helpers#48]
Meteor throws an error when using isSelected to select an HTML option.

Repro:
```
<option value="somevalue" {{ isSelected 'foo' 'foo'}}>Something</option>
```

<pre>
debug.js:41 Exception from Tracker recompute function: Error: Expected valid attribute name, '', null, or object
    at Object.Spacebars.attrMustache (http://localhost:3200/packages/spacebars.js?3c496d2950151d744a8574297b46d2763a123bdf:133:11)
    at http://localhost:3200/client/pages/profile/template.profile.js?26f38d9b14ba66d83872a8dde5951607b8f6cdbe:45:24
    at Object.Blaze._withCurrentView (http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:2043:12)
    at Blaze._HTMLJSExpander.def.visitAttributes (http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:1962:21)
    at HTML.TransformingVisitor.def.visitAttributes (http://localhost:3200/packages/htmljs.js?567eb96d5d22631c03d6aca6afa4c42f0d1295f2:202:44)
    at Blaze._HTMLJSExpander.def.visitAttributes (http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:1965:63)
    at Object.Blaze._expandAttributes (http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:1994:32)
    at Blaze._DOMMaterializer.def.visitTag.updateAttributes (http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:1469:35)
    at http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:1808:16
    at Object.Blaze._withCurrentView (http://localhost:3200/packages/blaze.js?efa68f65e67544b5a05509804bf97e2c91ce75eb:2043:12)
</pre>

I found the error in the debugger and poked around a bit. The reason this is failing is because Meteor assumes that an HTML string with a space-prefix is NOT a valid attribute name.

result
" selected"   // The string returned by isSelected
HTML.isValidAttributeName(result)   // The call within Meteor that is failing
false
HTML.isValidAttributeName("selected")  // Re-run in console without the space
true

**Comment by evenfrost** [comment]
I may be wrong but I believe `isSelected` is a kind of Spark legacy method because [boolean attributes work differently in Blaze nowadays](https://github.com/meteor/meteor/wiki/Using-Blaze#conditional-attributes-with-no-value-eg-checked-selectedonal-attributes-with-no-value-eg-checked-selected). So, in your case you can just use:
```handlebars
<option value="somevalue" selected="{{$eq 'foo' 'foo'}}">Something</option>

**Comment by serkandurusoy** [comment]
@vaughnkoch this has been fixed with PR #59 and will be available on the next release. Thanks for reporting.
